front_end/ui/legacy/components/object_ui/BindObjectPropertiesSection.ts

```diff
@@ -0,0 +1,2090 @@
+    const skipProto = true;
+    return ObjectPropertyTreeElement.populate(
+      this, this.object, skipProto, this.linkifier, this.emptyPlaceholder, this.propertiesMode, this.extraProperties,
+      this.targetObject);
+  }
+  // COHERENT_END
+}
+
+// Number of initially visible children in an ObjectPropertyTreeElement.
+// Remaining children are shown as soon as requested via a show more properties button.
+export const InitialVisibleChildrenLimit = 200;
+
+export class ObjectPropertyTreeElement extends UI.TreeOutline.TreeElement {
+  property: SDK.RemoteObject.RemoteObjectProperty;
+  toggleOnClick: boolean;
+  private highlightChanges: UI.UIUtils.HighlightChange[];
+  private linkifier: Components.Linkifier.Linkifier | undefined;
+  private readonly maxNumPropertiesToShow: number;
+  nameElement!: HTMLElement;
+  valueElement!: HTMLElement;
+  private rowContainer!: HTMLElement;
+  readOnly!: boolean;
+  private prompt!: ObjectPropertyPrompt | undefined;
+  private editableDiv!: HTMLElement;
+  propertyValue?: ObjectPropertyValue;
+  expandedValueElement?: Element | null;
+  constructor(property: SDK.RemoteObject.RemoteObjectProperty, linkifier?: Components.Linkifier.Linkifier) {
+    // Pass an empty title, the title gets made later in onattach.
+    super();
+
+    this.property = property;
+    this.toggleOnClick = true;
+    this.highlightChanges = [];
+    this.linkifier = linkifier;
+    this.maxNumPropertiesToShow = InitialVisibleChildrenLimit;
+    this.listItemElement.addEventListener('contextmenu', this.contextMenuFired.bind(this), false);
+    this.listItemElement.dataset.objectPropertyNameForTest = property.name;
+  }
+
+  // COHERENT_BEGIN
+  static async getPropsFromValue(value: SDK.RemoteObject.RemoteObject, propertiesMode: ObjectPropertiesMode = ObjectPropertiesMode.OwnAndInternalAndInherited) {
+    let properties, internalProperties = null;
+    switch (propertiesMode) {
+      case ObjectPropertiesMode.All:
+        ({ properties } = await value.getAllProperties(false /* accessorPropertiesOnly */, true /* generatePreview */));
+        break;
+      case ObjectPropertiesMode.OwnOnly:
+        ({ properties } = await value.getOwnProperties(true /* generatePreview */));
+        break;
+      case ObjectPropertiesMode.OwnAndInternalAndInherited:
+        ({ properties, internalProperties } =
+          await SDK.RemoteObject.RemoteObject.loadFromObjectPerProto(value, true /* generatePreview */));
+        break;
+    }
+
+    return { properties, internalProperties };
+  }
+
+  static async populate(
+    treeElement: UI.TreeOutline.TreeElement, value: SDK.RemoteObject.RemoteObject | undefined, skipProto: boolean,
+    linkifier?: Components.Linkifier.Linkifier, emptyPlaceholder?: string | null,
+    propertiesMode: ObjectPropertiesMode = ObjectPropertiesMode.OwnAndInternalAndInherited,
+    extraProperties?: SDK.RemoteObject.RemoteObjectProperty[],
+    targetValue?: SDK.RemoteObject.RemoteObject): Promise<void> {
```

Comment:
For me at least, makes it easier to follow these params
```suggestion
    treeElement: UI.TreeOutline.TreeElement,
    value: SDK.RemoteObject.RemoteObject | undefined, 
    skipProto: boolean,
    linkifier?: Components.Linkifier.Linkifier, 
    emptyPlaceholder?: string | null,
    propertiesMode: ObjectPropertiesMode = ObjectPropertiesMode.OwnAndInternalAndInherited,
    extraProperties?: SDK.RemoteObject.RemoteObjectProperty[],
    targetValue?: SDK.RemoteObject.RemoteObject): Promise<void> {
```

Reply:
Done this for you even this methods are from the original component :D